### PR TITLE
Perf/en 455

### LINKED ISSUE #6390: [Feature]: remove unused components
### How the project can be improved?

In several places we have imports of components that we do not use. Let's remove them to decrease the size of the bundle

### What are the acceptance criteria?

- [ ]

### Additional information

_No response_

### What version of Vue Storefront this feature can be implemented?

2.4.2

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Baroshem**
## Description

Closes #6390 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.

#### Changelog
- [ ] I have updated the Changelog ([V1](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md)) [v2](https://docs-next.vuestorefront.io/contributing/creating-changelog.html) and mentioned all breaking changes in the public API.
- [ ] I have documented all new public APIs and made changes to existing docs mentioning the parts I've changed so they're up to date.

#### Tests
- [ ] I have written test cases for my code
- [ ] I have tested my Pull Request on production build and (to my knowledge) it works without any issues
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

> I tested manually my code, and it works well with both:
- [ ] Default Theme
- [ ] Capybara Theme

#### Code standards
- [ ] My code follows the code style of this project.

- [ ] I have followed [naming conventions](https://github.com/kettanaito/naming-cheatsheet)

#### Docs
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: packages/commercetools/theme/pages/Product.vue [hunk 1]
```diff
@@ -266,12 +262,20 @@ export default {
...
     // const breadcrumbs = computed(() => productGetters.getBreadcrumbs ? productGetters.getBreadcrumbs(product.value) : props.fallbackBreadcrumbs);
-    const productGallery = computed(() => productGetters.getGallery(product.value).map(img => ({
-      mobile: { url: img.small },
-      desktop: { url: img.normal },
-      big: { url: img.big },
-      alt: product.value._name || product.value.name
-    })));
+    // const productGallery = computed(() => productGetters.getGallery(product.value).map(img => ({
+    //   mobile: { url: img.small },
+    //   desktop: { url: img.normal },
```
Comment (dawid-ziobro) [inline packages/commercetools/theme/pages/Product.vue:267]: Commented code in this file

File: packages/commercetools/theme/pages/Product.vue [hunk 2]
```diff
@@ -266,12 +262,20 @@ export default {
...
+    //   big: { url: img.big },
+    //   alt: product.value._name || product.value.name
+    // })));
+    // console.log(productGallery);
+    const productGallery = ref([{
+      mobile: { url: '/homepage/productB.webp' },
+      desktop: { url: '/homepage/productB.webp' },
+      big: { url: '/homepage/productB.webp' },
+      alt: 'dupa'
```
Comment (dawid-ziobro) [inline packages/commercetools/theme/pages/Product.vue:276]: Nice `alt` :D

**Review by dawid-ziobro — APPROVED** [review]

**Comment by PiotrGrzywa** [comment]
Those changes will be manually ported to new commercetools repo.

**Comment by PiotrGrzywa** [comment]
Those changes are included in this PR:
https://github.com/vuestorefront/commercetools/pull/108

**Pull request closed without merging**